web/js/tasks.js

```diff
@@ -590,6 +590,7 @@ var TaskPanel = Ext.extend(Ext.Panel, {
                     //create a new template record
                     var newTemplate = new templateRecord();
                     newTemplate.set('name', task.get('text'));
+                    newTemplate.set('text', task.get('text'));
```

Comment:
Don't do that. As things are now, `task.get('text')' contains the template name, we are not interested in presenting that name in the description for tasks created from this template, users will always have to remove it.